docker-compose.yml

```diff
@@ -5,6 +5,7 @@ services:
         image: crossdock/crossdock
         dns_search: .
         links:
+            - redis
```

Comment:
Lets move this to the last link, the primary containers are the test participants.